packages/frontend-2/components/dashboard/Webflow/Wrapper.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+  <section v-if="!error">
+    <h2 class="text-heading-sm text-foreground-2 mb-4">Blog</h2>
+    <div class="grid grid-cols-1 sm:grid-cols-2 md:grid-cols-3 lg:grid-cols-4 gap-4">
+      <DashboardWebflowCard
```

Comment:
Maybe we should keep this naming a bit more ambiguous, so we can reuse/repurpose them?